### PR TITLE
FEAT-#2223: Add pydocstyle check for modin/pandas folder

### LINKED ISSUE #2223: Convert docstrings in low level APIs in modin/pandas folder to numpydoc style.
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. What kind of performance improvements would you like to see with this new API?

### PR DESCRIPTION
**Pull request by anmyachev**
## What do these changes do?

- [x] commit message follows format outlined [here](https://modin.readthedocs.io/en/latest/contributing.html)
- [x] passes `flake8 modin`
- [x] passes `black --check modin`
- [x] signed commit with `git commit -s` 
- [x] Resolves #2223 
- [ ] tests added and passing

### CODE REVIEW

File: modin/pandas/utils.py [hunk 1]
```diff
@@ -38,12 +62,13 @@ def from_pandas(df):
...
 def from_arrow(at):
-    """Converts an Arrow Table to a Modin DataFrame.
+    """
+    Convert an Arrow Table to a Modin DataFrame.
 
     Parameters
     ----------
-        at : Arrow Table
-            The Arrow Table to convert from.
+    at : Arrow Table
```
Comment (YarShev) [inline modin/pandas/utils.py:70]: As I understand, docstrings are edited automatically by `pydocstyle`. Why does `pydocstyle` remove space between parameter name and sign ":" in some cases, but sometimes the one doesn't remove it?
```python
example 1
at : Arrow Table # a space is not removed
example 2
val: object # a space is removed
```

File: modin/pandas/utils.py [hunk 2]
```diff
@@ -68,15 +93,14 @@ def is_scalar(obj):
 
     Parameters
     ----------
-    val : object
+    val: object
```
Comment (YarShev) [inline modin/pandas/utils.py:96]: See previous comment.

File: modin/pandas/utils.py (hunk 1 above)
Comment (anmyachev) [reply modin/pandas/utils.py:70]: Because it's done manually. `pydocstyle` does not automatically edit docstrings.

Comment (YarShev) [reply modin/pandas/utils.py]: Now I've got that.

**Comment by YarShev** [comment]
@anmyachev , LGTM!

**Review by YarShev — APPROVED** [review]

**Comment by anmyachev** [comment]
thanks @YarShev !

**Pull request merged**